src/gui/CMakeLists.txt

```diff
@@ -47,14 +42,21 @@ SET(LMMS_SRCS
 	gui/ClipView.cpp
 	gui/TrackView.cpp
 
+	gui/dialogs/AboutDialog.cpp
+	gui/dialogs/ColorChooser.cpp
```

Comment:
I don't see a move for this.

Reply:
Opens in external window as mentioned on Discord